### PR TITLE
fix cmake includes for ipp static standalone libraries

### PR DESCRIPTION
**Pull request by diablodale**
fixes opencv/opencv#19282 with details in that issue.

TLDR: fixes opencv static libraries with windows IPP standalone static library list in `OpenCVModules.cmake` 

This has been tested locally on the current `4.5` branch (specifically 4.5.1). I have not tested it on the legacy `3` branch.
I diff'd the `OpenCVModules.cmake` file changed in this PR and it is the same in both `4.5.1` and `3` branches.
Therefore, it should apply cleanly to the legacy branch.
Caution: I have no knowledge about the viability of this change on legacy `3` opencv, its static support, its IPP support, etc. I do not have the os, compilers, harness, etc for that legacy setup.

### Pull Request Readiness Checklist

- [X] I agree to contribute to the project under Apache 2 License.
- [X] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [X] The PR is proposed to proper branch
- [X] There is reference to original bug report and related work
- [X] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by asmorkalov** [comment]
@alalek Could you comment the solution?

**Comment by diablodale** [comment]
> @alalek Could you comment the solution?

Hi, I'm the OP. Alalek and I had a good conversation in the issue for this PR https://github.com/opencv/opencv/issues/19282
In it, I suggested that the core OpenCV team use their Intel contacts to communicate directly with the Intel IPP team...and have that team create standard cmake find/version/package handlers...and that team submit them to kitware to make part of the official cmake distribution. After that work, OpenCV could then use the official method. (naturally, this is a multi-month process)

**Comment by diablodale** [comment]
Hi. I'm going to close this PR for now. The linked issue https://github.com/opencv/opencv/issues/19282 has good discussion and this PR no longer applies.
